src/pubtools/_marketplacesvm/tasks/community_push/command.py

```diff
@@ -502,6 +502,7 @@ def _data_to_upload(
                         "marketplace": storage_account,
                         "push_item": pi,
                         "ami_version_template": ami_version_template,
+                        "custom_tags": {"billing_type": pi.type},
```

Comment:
Will this always be available? I think the there's some initial pushes for community that don't appear to have type in the clouds.json.

Reply:
Fixed, PTAL again